terra_notebook_utils/table.py

```diff
@@ -39,6 +39,26 @@ def fetch_drs_url(table: str, file_name: str):
         raise ValueError(f"Expected DRS url in {table} for {file_name}, got {val} instead.")
     return val
 
+def list_entities(ent_type):
+    resp = fiss.fapi.get_entities_with_type(WORKSPACE_GOOGLE_PROJECT, WORKSPACE_NAME)
+    resp.raise_for_status()
+    for ent in resp.json():
+        if ent['entityType'] == ent_type:
+            yield ent
+
+def delete_entities(entities: list):
+    ents = [dict(entityType=e['entityType'], entityName=e['name'])
+            for e in entities]
+    resp = fiss.fapi.delete_entities(WORKSPACE_GOOGLE_PROJECT, WORKSPACE_NAME, ents)
+    resp.raise_for_status()
+
+def delete_table(ent_type: str):
+    delete_entities([e for e in list_entities(ent_type)])
```

Comment:
Will we ever need to batch this?

Reply:
Future work, see: https://github.com/DataBiosphere/terra-notebook-utils/issues/21